Source/Core/Common/Config/Config.h

```diff
@@ -118,4 +114,10 @@ void SetBaseOrCurrent(const ConfigInfo<T>& info, const T& value)
   else
     Set<T>(LayerType::CurrentRun, info, value);
 }
+
+template <typename T>
+bool Delete(LayerType layer_type, const ConfigInfo<T>& info)
```

Comment:
FIXME: Something (that's not a fault of this PR) that I've noted is that config deletions won't get saved because of the Load in the configloaders here: https://github.com/leoetlino/dolphin/blob/8299814fd49a36ac56aa9f8a677028f06610e241/Source/Core/Core/ConfigLoaders/BaseConfigLoader.cpp#L122